client/state/plugins/installed/actions.js

```diff
@@ -173,9 +173,16 @@ export function activatePlugin( siteId, plugin ) {
 		};
 
 		const successCallback = ( data ) => {
-			dispatch( { ...defaultAction, type: PLUGIN_ACTIVATE_REQUEST_SUCCESS, data } );
-			dispatch( handleDispatchSuccessCallback( defaultAction, data ) );
-			afterActivationCallback( undefined, data );
+			// Add shim for wp/v2 API response format
+			const normalizedData = {
```

Comment:
The response is different in `wp/v2` so we need to normalize it otherwise the UI won't respond correctly

Reply:
Looks fine. I hope this doesn't affect the non-A4A environments.

Reply:
Looks fine. I hope this doesn't affect the non-A4A environments.